test/common.h

```diff
@@ -4,7 +4,7 @@
 
 #include "manifest.h"
 
-#define memdup(x) (g_memdup(x, sizeof(*x)))
```

Comment:
This seems to break the test suite (test/nbd-test).

Reply:
Should be fixed by the new approach.